packages/Babylonian-UI.package/BPProbeMorph.class/instance/newLabelMorph.st

```diff
@@ -0,0 +1,31 @@
+private - ui
+newLabelMorph
+	
+	| container labelSource labelFieldMorph |
+	container := Morph new
+		color: Color transparent;
+		changeTableLayout;
+		listDirection: #topToBottom;
+		vResizing: #shrinkWrap;
+		hResizing: #spaceFill;
+		cellPositioning: #leftCenter;
+		borderWidth: 1;
+		borderColor: Color black;
+		layoutInset: 1;
+		yourself.
```

Comment:
i'm okay with leaving out the border color or providing it as an argument for the sake of extracting this part as it is exactly like the expression morph container part in `newExpressionMorph`

Reply:
i'd also argue the rest is extractable but OK in this message if the rest stands there as it does

Reply:
extracted it into newContainerMorph and removed the border